basics/Strings examples/find multiple occurances 2 easy.py

```diff
@@ -0,0 +1,13 @@
+str = input("Type the string")
+ch=input("Type the occurance needed to be found:")
+i=0
+
+
+while i <len(str):
+    start=i
+    ins=str.find(ch,i,len(str))
+    if ins==-1:
+        break
+    else:
+        print(ins)
+        i=ins+1
```

Comment:
This too finds `asa` from `asasa` twice. To fix it, replace `i = ins + 1` with `i = ins + len(ch)` (and please switch to better variable names).